protocol/daemons/server/types/liquidations/daemon_liquidation_info.go

```diff
@@ -0,0 +1,117 @@
+package types
+
+import (
+	"sync"
+
+	clobtypes "github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+	satypes "github.com/dydxprotocol/v4-chain/protocol/x/subaccounts/types"
+)
+
+// DaemonLiquidationInfo maintains the list of subaccount ids to be liquidated
+// in the next block. Methods are goroutine safe.
+type DaemonLiquidationInfo struct {
+	sync.Mutex                                       // lock
+	blockHeight               uint32                 // block height of the last update
```

Comment:
Are all of the liquidatable / negative TNC subaccounts read from the same block height (which is the same as this `blockHeight` field)?

Reply:
yes - I will be updating the liquidation daemon to make height based queries so everything is read from the same height